### DISCUSSION TITLE
Discuss a11y (accessibility)

### DISCUSSION THREAD

**Discussion by avidrucker** [knowledge-for-good/knowledgeforgood#8] [General]
Description text TBD...

**Comment by avidrucker** [comment]
I want icons for everything in my life (eg. emoji, awesomefonts, etc.), but why?

**Reply by seekanddefine** [reply]
Incorporating visuals is very important and show to enhance memory retention.

**Comment by avidrucker** [comment]
Could this be represented as a tree, simply, and in an easily transferrable (ie. text) format? https://andreasbm.github.io/web-skills/?compact

**Reply by seekanddefine** [reply]
I would love to see this as a tree. Perhaps the tree structure is one that is created as the learner learns?

**Reply by avidrucker** [reply]
Are you aware of any good tools for displaying, creating, sharing tree diagrams @seekanddefine ?
